worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -79,6 +80,13 @@ private[celeborn] class Worker(
       conf.workerRpcPort != 0 && conf.workerFetchPort != 0 &&
       conf.workerPushPort != 0 && conf.workerReplicatePort != 0),
     "If enable graceful shutdown, the worker should use stable server port.")
+  if (gracefulShutdown) {
+    val recoverRoot = new File(conf.workerRecoverPath)
+    if (!recoverRoot.exists()) {
+      logInfo(s"Recover root path ${conf.workerRecoverPath} does not exists, create it first.")
+      recoverRoot.mkdirs()
```

Comment:
We should support mkdir -P? and also have a try catch here?

Reply:
ok, i will update later

Reply:
> We should support mkdir -P? and also have a try catch here?

I think mkdirs() is equivalent to mkdir -P